app/controllers/admin/api/accounts_controller.rb

```diff
@@ -258,4 +249,21 @@ def account_plan
   def billing_params
     @billing_params ||= params.permit(:monthly_billing_enabled, :monthly_charging_enabled)
   end
+
+  private
+
+  def find_buyer_account
```

Comment:
imo this method needs tests 😄

Reply:
it's a private method so only integration ones are allowed 🤒

Reply:
ok but then the integration should test the response with this params 😄

Reply:
🍤 sure :)